config.toml.dist

```diff
@@ -135,6 +129,55 @@ generate_rsa_command = ["openssl", "genrsa", "2048"]
 
 #postmark_token = ""
 
+################################################################
+# Access control
+
+# The lists in this section all take literal string values, but also filenames
+# by prefixing a path with `@`. These files must contain one value per line,
+# but may contain empty lines or comments starting with `//` or `#`.
+#
+# Note that files are only loaded once during startup, and the broker must be
+# restarted after updating them.
+
```

Comment:
It would be helpful to ass a newline here to separate the introduction setting from the next setting.